bootstrap/cli/remote/session.py

```
@@ -555,6 +555,26 @@ def _check_duplicate_server_ids(root: Path, session: Session, issues: list) -> N
+    remote; the publisher skips re-uploading them, so their blobs are not
+    required locally (a metadata-only sync never downloads blobs).
+    """
+    from bootstrap.remote.generation import GenerationStore
+    from bootstrap.remote.head import ChannelHeadStore
+
+    try:
+        head = ChannelHeadStore(root)._safe_get_head(channel)
+        if not head or not head.generation_hash:
+            return set()
+        gen = GenerationStore(root).load(head.generation_hash)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)